### DISCUSSION TITLE
Cannot pass data scan rules yaml file in DataplexCreateOrUpdateDataQualityScanOperator

### DISCUSSION THREAD

**Discussion by sukalpv** [apache/airflow#36872] [Q&A]
### Apache Airflow Provider(s)

google

### Versions of Apache Airflow Providers

apache-airflow-providers-google === 10.13.0

### Apache Airflow version

2.6.3

### Operating System

Ubuntu 20.04.6 LTS

### Deployment

Google Cloud Composer

### How to reproduce

Is it possible to pass data scan rules in yaml file to DataplexCreateOrUpdateDataQualityScanOperator or terraform? There is no documentation which states how can it be done.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by Taragolis** [comment]
Converting it into discussion - it's not really an issue, it's just a question.

Just wondering did you check this pages before, maybe you could find your answer here
- https://airflow.apache.org/docs/apache-airflow-providers-google/stable/operators/cloud/dataplex.html#create-or-update-a-data-quality-scan
- https://airflow.apache.org/docs/apache-airflow-providers-google/stable/_api/airflow/providers/google/cloud/operators/dataplex/index.html#airflow.providers.google.cloud.operators.dataplex.DataplexCreateOrUpdateDataQualityScanOperator

**Reply by sukalpv** [reply]
Yes, already checked those links. The rules can be defined in the body but it is not possible to pass them as file reference. 
```
EXAMPLE_DATA_SCAN.data_quality_spec = DataQualitySpec(
    {
        "rules": [
            {
                "range_expectation": {
                    "min_value": "0",
                    "max_value": "10000",
                },
                "column": "value",
                "dimension": "VALIDITY",
            }
        ],
    }
)
```
However, gcloud command line allows to pass it as file reference
https://cloud.google.com/dataplex/docs/use-auto-data-quality#gcloud
```
gcloud dataplex datascans create data-quality DATASCAN \
--location=LOCATION \
--data-quality-spec-file=DATA_QUALITY_SPEC_FILE \
--data-source-entity=DATA_SOURCE_ENTITY
| --data-source-resource=DATA_SOURCE_RESOURCE
```

**Reply by Taragolis** [reply]
`apache-airflow-providers-google` use [Cloud SDK](https://cloud.google.com/python/docs/reference), and not CLI.

**Reply by sukalpv** [reply]
@Taragolis thanks, it helps. Could you please help me if passing jinja templates works in DataplexCreateOrUpdateDataQualityScanOperator. What I am trying to do,
```
data_quality_scan.data_quality_spec = DataQualitySpec({
        'rules': [
            {
                'table_condition_expectation': {
                    'sql_expression': f'''MAX(event_ts) > {{ ds }} 23:00:00 UTC '''
                },
                'dimension': 'VALIDITY',
            },
        ],
    })
```
{{ ds }} is not getting replaced. The doc says jinja templates works for body though. Thanks 🙏
